src/api/v1beta1/feature_flags.go

```diff
@@ -47,17 +48,17 @@ var (
 
 // FeatureDisableActiveGateUpdates is a feature flag to disable ActiveGate updates.
 func (dk *DynaKube) FeatureDisableActiveGateUpdates() bool {
-	return dk.Annotations[annotationFeatureDisableActiveGateUpdates] == "true"
+	return dk.getFeatureFlagRaw(annotationFeatureDisableActiveGateUpdates) == "true"
```

Comment:
Out of scope of this changeset, but food for thought: `[strconv.ParseBool()](https://pkg.go.dev/strconv#hdr-Numeric_Conversions)`

Reply:
`ParseBool` has an error return so, we would need to handle that, so it wouldn't really improve the code 😅

Reply:
It would improve the user experience of these flags in that you could specify boolean values in a predictable, documented way. For example, if you specify `"True"` and use `.ParseBool()`, it will be interpreted as boolean `true`. How will the current code interpret `"True"`? :)

No big deal though and out of scope of this PR as I mentioned.